### ISSUE TITLE
Review issues

### ISSUE THREAD

**Issue by robinwhittleton** [dajare/walter-bagehot_the-english-constitution#1]
**general**
- run build-manifest again, it should add a piece of missing information about alt text. If not, can you make sure you’re using the current version of the toolset?

**content.opf**
- shortdesc: Do we need a timestamp? e.g. “British governance in the Victorian era”

**chapter-1**
- Change the cells in the `thead` from `td` to `th`. That’ll introduce some default styling that you may need to undo, you’ll also need to rewrite a couple of selectors in local.css

**chapter-3**
- `Geoge III` is missing an `r`
- We have `<em>un</em>-royal`, but compare to later `unroyal` and chapter 4’s `<em>un</em>popular`. I think we should remove the dash.

**chapter-5**
- `happliy` -> `happily`

**chapter-6**
- You use `personal-name` for “A. B.” earlier in the chapter which is correct, then change to `given-name` for the other name `abbr`s later on

**chapter-9**
- A couple of greek words `βουλή` and `ἀγορά` that need italics and lang attributes

**endnotes**
- There are a couple of `<i>`s inside `<cite>`s, that I think can just be removed.

**local.css**
- picky, but the table margins can be expressed more concisely as `margin: 1em auto`

**Comment by dajare** [comment]
Thanks for the review, Robin.

For the name semantics, I also changed `<abbr epub:type="z3998:personal-name">A.</abbr>`, although my thinking on that one was that in the earlier `A. B.` (`personal-name`), the `A.` was the given name. I can switch that back if you think that's better. Or, if it's uncertain, we can keep the current `personal-name`.

Not sure how I missed those Greek words - I guess they just looked "normal". `:)`

I think this is all done now. Could you take another look?

**Comment by robinwhittleton** [comment]
To double-check: you’re happy with the table styling after changing the cells to be `th`s? (I’m on my lunchbreak so can’t easily check). If so I’ll let Alex know this is ready.

**Comment by dajare** [comment]
> To double-check: you’re happy with the table styling after changing the cells to be `th`s? (I’m on my lunchbreak so can’t easily check). If so I’ll let Alex know this is ready.

Sure - looks like this for me:

<img width="523" height="330" alt="Image" src="https://github.com/user-attachments/assets/ae06fc56-c21e-477c-a089-aadcab9c2fff" />

which works well, imo. Thanks!
